src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -1276,7 +1277,7 @@ export default class Aircraft {
         this.fms.followSID(route.toUpperCase());
 
         const readback = {
-            log: `cleared to destination via the ${sidId} departure, then as filed`,
+            log: `cleared to destination via the ${sidIdOutput} departure, then as filed`,
```

Comment:
Is there a reason we would need it stored as a variable?  Believe it or not, I'm inclined to say we should just put this inline. Or, better yet, is this a value that could be obtained from the `StandardRouteCollection`?